usage.md

```diff
@@ -0,0 +1,134 @@
+# Usage
+
+## resolve-tags plugin
+
+See the [resolve-tags plug-in guide](https://github.com/grafeas/kritis/blob/master/cmd/kritis/kubectl/plugins/resolve/README.md)
+
+## Resource Reference
+
+Installing Kritis, creates a number of resources in your cluster. Mentioned below are important ones.
+
+| Resource Name | Resource Kind | Description |
+|---------------|---------------|----------------|
+| kritis-validation-hook| ValidatingWebhookConfiguration | This is Kubernetes [Validating Admission Webhook](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers) which enforces the policies. |
+| imagesecuritypolicies.kritis.grafeas.io | crd | This CRD defines the image security policy kind ImageSecurityPolicy.|
+| attestationauthorities.kritis.grafeas.io | crd | The CRD defines the attestation authority policy kind AttestationAuthority.|
+| tls-webhook-secret | secret | Secret required for ValidatingWebhookConfiguration|
+
+### kritis-validation-hook
+
+The validating admission Webhook runs a https service and a background cron job.
+The webhook, runs when pods and deployments are created or updated in your cluster.
+To view webhook, run
+
+```shell
+kubectl describe ValidatingWebhookConfiguration kritis-validation-hook
+```
+
+The cron job runs hourly to continuously validate and reconcile policies. It adds labels and annotations to pods out of policy.
```

Comment:
nit: make might sense to mention the `kubectl exec -l label=kritis-validation-hook -- /kritis/kritis-server --run-cron` command to manually exec the cron job as well as the `kubetl get pods -l kritis.grafeas.io/invalidImageSecPolicy=invalidImageSecPolicy` command to see the invalid pods after the cron run

Reply:
Done.